_digest/october-2020/making-the-most-out-of-your-morning.mdx

```diff
@@ -1,76 +1,28 @@
 ---
-title: Making the most out of your morning: Establishing Morning Habits en route to medical school 
+title: 'Making the most out of your morning: Establishing Morning Habits en route to medical school'
```

Comment:
I think these should be backticks (above tab key/left of numbers), not single quotation marks. Alternatively, change the `:` to a dash `... your morning - Establishing Morning Habits ...`

Reply:
@hirundinidae I used single quotation marks when my "title" part of the front matter also had a colon. curious, why should it be backticks instead?

Reply:
I think single quotes actually works as well, I was mistaken + going off of Sharon's comment 
> please change it to title: `Making the most out of your morning: Establishing Morning Habits en route to medical school' ...

which mixed backtick + single quote, thought it meant to be backticks.